app/views/user_mailer/new_contacts_digest.html.erb

```diff
@@ -17,4 +17,5 @@ PLEASE NOTE: Guests and hosts have NOT been vetted by the campaign, by<br>
 volunteers, or by anyone else. It is your responsibility to contact<br>
 potential hosts and visitors in advance and decide whether you are<br>
 comfortable and safe staying with them.<br><br>
-You can add, edit or delete your host offers anytime here: <%= link_to "HillaryBNB - Host", edit_hosting_url(@hosting) %>
+You can add, edit or delete your host offers anytime here: <%= link_to "HillaryBNB - Host", edit_hosting_url(@hosting) %><br>
+To stop receiving emails from HillaryBNB, you can <%= link_to "unsubscribe", unsubscribe_url() %>.
```

Comment:
Once your route is in place, it should be `unsubscribe_user_url`. Note that it's not considered good form to expose the real user id since it becomes very easy to spoof these requests. It could be OK if the account is password protected and they will need to log in to unsubscribe. Otherwise you could consider adding a column to user and generating a random token for each user that would be used in this url.